samples/saga/ravendb-sequence-generator/Version_5/Sample/SequenceSagaData.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using NServiceBus.Saga;
+
+#region OrderSagaDataRavenDB
+
+public class SequenceSagaData : IContainSagaData
```

Comment:
You could derive from `ContainSagaData` class which already implements `IContainSagaData`.